### ISSUE TITLE
very bad results when training on customized English data 

### ISSUE THREAD

**Issue by deeptek012** [JaidedAI/EasyOCR#593]
Thank you so much for the project. It helps a lot! 

But when we want to train the customized data, we could only see significant performance decline comparing to the default model (english_g2.pth). Could you help to see what's going on? 

We followed the [Instruction](https://github.com/JaidedAI/EasyOCR/blob/master/custom_model.md). We first generated ~2M images using the customized content (all English with the same chars and orders as in the default config file). Second, we ran training by the following configs.
1. Retraining from scratch ( freeze_FeatureFxtraction is False, no saved model). We tried different batch sizes (32, 64, 256). Each case we trained about 600K iterations. The training accuracy reaches about 80%, training loss reduced to 0.2. But the best_accuracy model performed very bad. The testing accuracy dropped from 36% (english_g2) to around 1%. 
2. Finetuning from english_g2. (freeze_FeatureExtraction is True). We tried batch size 32, 150K iterations. The training acc is around 60%. The testing is worse and worse as we fine tune more iterations. (36% original -> 22% -> 2%)

We wonder what could go wrong. Can you please help?

**Comment by mrkrcc** [comment]
any updates? @deeptek012
